integreat_cms/cms/templates/dashboard/_todo_dashboard_widget.html

```diff
@@ -13,7 +13,7 @@
                 Every day you can find a list of ideas on how you can improve the content of your pages
             {% endblocktranslate %}
         </p>
-        {% if perms.cms.change_page %}
+        {% if perms.cms.change_page and request.region.default_language is not None %}
```

Comment:
Or maybe we should just check if the relevant keys are in the request context at all?

Reply:
No, since this template is only shown in the region dashboard, we can assume that `request.region` is a valid region object.

But I agree with the others that `is not None` is not required here, since it's either a `Language` object or `None` and a `Language` object can never evaluate to a Falsey value, so `False` and `None` are indeed equivalent in this context.
```suggestion
        {% if perms.cms.change_page and request.region.default_language %}
```

Reply:
I meant we could make it data dependent and leave it to [the view](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/views/dashboard/dashboard_view.py#L51-L83) to populate the context with variables like `unreviewed_pages`, and if they are missing, just don't display the widget. That would also mean we don't have essentially the same check whether the Textlab API is enabled globally and for that region in both the view and [the template rendering it](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/templates/dashboard/_todo_dashboard_widget.html#L26-L28)

Reply:
Ah ok, sorry for the misunderstanding.
Hmm, I guess it would be a bit complicated to determine the difference between an empty variable and an unset variable in the template...
E.g. `{% if pages_under_hix_threshold %}` will be `True` only if pages with a low HIX value exist and `False` if Textlab is disabled or if it's enabled but all pages have a sufficient HIX value... But we want to hide the row if it's disabled and show "All good" if not, so I think it makes sense to keep the current conditions.
It would be possible to determine the difference e.g. in a custom template tag, but in my opinion this would make it more complicated and not easier. Or we could add new context variables for each box, e.g. `show_low_hix_row` and set this from the view...